### ISSUE TITLE
Security Vulnerabilities in lodash 4.17.15

### ISSUE THREAD

**Issue by flymithra** [lodash/lodash#4943]
Hello, made some checks for security vulnerabilities and got that report:

  CRITICAL
  -----
  CVE-2019-10744, CVE-2018-16487                    lodash.uniq:4.5.0  	(package-lock.json)
  CVE-2019-10744, CVE-2018-16487                    lodash.some:4.6.0  	(package-lock.json)
  CVE-2019-10744, CVE-2018-16487                    lodash.defaults:4.2.0  	(package-lock.json)
  CVE-2019-10744, CVE-2018-16487                    lodash.sortby:4.7.0  	(package-lock.json)
  CVE-2019-10744, CVE-2018-16487                    lodash.isarguments:3.1.0  	(package-lock.json)
  CVE-2019-10744, CVE-2018-16487                    lodash.clonedeep:4.5.0  	(package-lock.json)
  CVE-2019-10744, CVE-2018-16487                    lodash.assign:4.2.0  	(package-lock.json)
  CVE-2019-10744                                    lodash:4.17.15  	(package-lock.json)
  CVE-2019-10744, CVE-2018-16487                    lodash.camelcase:4.3.0  	(package-lock.json)
  CVE-2019-10744, CVE-2018-16487                    lodash.assign:3.2.0  	(package-lock.json)

I googled it, but everywhere people say, that I should upgrade lodash to 4.17.15, but I'm using it. 

Any thoughts how can I fix them?

**Comment by FelixRilling** [comment]
Is it possible that a different dependency depends on a vulnerable version of lodash? That would explain why the package-lock.json is mentioned; `npm ls` might show where it is used.

**Comment by flymithra** [comment]
I use ```npm dedupe```, so it can't be another dependency

btw, here's my ```npm ls```


```

$npm ls | grep lodash@

│ │ ├── lodash@4.17.15 deduped
│ │ ├── lodash@4.17.15 deduped
│     └── lodash@4.17.15 deduped
│ └── lodash@4.17.15 deduped
│ │ ├── lodash@4.17.15 deduped
│ │ │ ├── lodash@4.17.15 deduped
│ │ ├── lodash@4.17.15 deduped
│ │ └── lodash@4.17.15 deduped
│ │ └── lodash@4.17.15 deduped
│ │ ├── lodash@4.17.15 deduped
│ ├── lodash@4.17.15 deduped
│ │ │ ├── lodash@4.17.15 deduped
│ │ │ │ └── lodash@4.17.15 deduped
│ │ └── lodash@4.17.15 deduped
│ │ ├── lodash@4.17.15 deduped
│ │ └── lodash@4.17.15 deduped
│ │ │ └── lodash@4.17.15 deduped
│ │ │ └── lodash@4.17.15 deduped
│ │ ├── lodash@4.17.15 deduped
│ ├── lodash@4.17.15 deduped
│ │ ├── lodash@4.17.15 deduped
│   │ │ ├── lodash@4.17.15 deduped
│   │   │ │ └── lodash@4.17.15 deduped
npm│   │ │ ├── lodash@4.17.15 deduped
 ERR!├── lodash@4.17.15
 peer dep missing: ajv@>=5.0.0, required by ajv-errors@1.0.1
npm ERR!│ │   ├── lodash@4.17.15 deduped
 peer dep missing: rollup@< 0.59.0, required by rollup-plugin-json@2.3.1
npm ERR! peer dep missing: ajv@>=5.0.0, required by ajv-errors@1.0.1
│ │ ├── lodash@4.17.15 deduped
│   ├── lodash@4.17.15 deduped
│ │ ├── lodash@4.17.15 deduped
│   │ └── lodash@4.17.15
│ │ └── lodash@4.17.15 deduped
│ │ │ │ ├── lodash@4.17.15 deduped
│ │ │ ├── lodash@4.17.15 deduped
│ │ ├── lodash@4.17.15 deduped
  │ ├── lodash@4.17.15 deduped
```

**Comment by FelixRilling** [comment]
Yeah, then that's most likely not the issue - which tool generated the vulnerability list in the original post?

**Comment by falsyvalues** [comment]
Hi @flymithra!
Please verify your dependencies, latest available version is [v4.17.20](https://www.npmjs.com/package/lodash).
